functests/4_latency/latency.go

```diff
@@ -413,3 +425,20 @@ func extractLatencyValues(testName string, exp string, node *corev1.Node) string
 
 	return latencies[1]
 }
+
+func removeLogfile(node *corev1.Node, logName string) {
+
+	cmd := []string{"cat", fmt.Sprintf("/rootfs/var/log/%s.log", logName)}
+	out, err := nodes.ExecCommandOnNode(cmd, node)
+	if err != nil {
+		testlog.Error(err)
+	}
+	fmt.Printf("the test's output log is: \n %s \n", out)
```

Comment:
In case of test failure, the log will be printed anyway, so now it will be printed twice?
Also, nothing was printed in case of success, now the output will be printed anyway. is that the desired behavior?

Reply:
"In case of test failure, the log will be printed anyway, so now it will be printed twice?"
will fix that 
  "Also, nothing was printed in case of success, now the output will be printed anyway. is that the desired behavior?"
Following the discussion with @cynepco3hahue, no we don't, we need it outputted to the console just in case of failure

Reply:
@Tal-or after checking the first part , the log is printed only once